docs/distributions/manjaro/faq.md

```diff
@@ -1,30 +1,60 @@
+# Migrating to new kernel
+
+Run this in your terminal:
+
+```sh
+cat << EOF | sudo tee -a /etc/pacman.conf
+[manjaro-mact2]
+SigLevel = Never
+Server = https://mirror.funami.tech/$repo/os/$arch
+EOF
+sudo pacman -Syu linux516-t2 linux516-t2-headers dkms apple-bce-dkms-git apple-ibridge-dkms-git apple-bcm-wifi-firmware
+sudo pacman -R linux57-mbp 
```

Comment:
does pacman allow you to pacman -R package1 package2 etc on one line?

Reply:
If at least one of them isn't installed, it fails completely and deletes no package. It's a workaround.